mm2src/mm2_core/src/mm_ctx.rs

```diff
@@ -221,7 +221,7 @@ impl MmCtx {
 
     pub fn is_watcher(&self) -> bool { self.conf["is_watcher"].as_bool().unwrap_or_default() }
 
-    pub fn use_watchers(&self) -> bool { self.conf["use_watchers"].as_bool().unwrap_or_default() }
+    pub fn use_watchers(&self) -> bool { self.conf["use_watchers"].as_bool().unwrap_or(true) }
```

Comment:
Is it ok this is not documented [here](https://developers.komodoplatform.com/basic-docs/atomicdex/atomicdex-setup/configure-mm2-json.html#mm2-json)

@smk762 @shamardy

Reply:
It's better to create an issue to be sure it would not be forgotten

Reply:
Done, [here](https://github.com/KomodoPlatform/atomicDEX-API/issues/1866)